doc/api-extensions.md

```diff
@@ -415,3 +415,31 @@ same and between storage pools.
 
 ## operation\_description
 Adds a "description" field to all operations.
+
+## clustering
+Clustering API for LXD.
+
+This includes the following new endpoints:
+
+* `GET /1.0/cluster`
+* `DELETE /1.0/cluster` (see [RESTful API](rest-api.md) for details)
```

Comment:
Should be "UPDATE /1.0/cluster" and also be used to initialize the cluster or join an existing cluster (see other comments).

Reply:
Added.